.github/workflows/static-code-analysis.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Run PHP-CS-Fixer
         uses: docker://jakzal/phpqa:php7.3-alpine
         with:
-            args: php-cs-fixer fix --dry-run --diff-format udiff -vvv
+            args: php-cs-fixer fix --dry-run -vvv
```

Comment:
Also, I'd like to keep `--diff` option as well to see the actual diff in the CI. Otherwise, it does not show the diff by default IIRC

Reply:
Resolved in [9bdadcf](https://github.com/php-translation/symfony-storage/pull/58/commits/9bdadcf354c30f03e35f1f737956297fc2e8ee3d)